script.js

```diff
@@ -64,59 +64,91 @@ function resetGame() {
     pumpkinMove.pause();
     reaperMove.cancel();
     grimReaperZone.removeEventListener('mouseleave', moveReaper);
+
 }
 
 // code here to activate win modal displaying
 function win() {
     winModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
     resetGame();
 }
 
 // code here to activate lose modal displaying
 function lose() {
     loseModal.style.display = "block";
+    clearInterval(timer);
+    timerRunning = false;
     resetGame();
 }
 
 winModal.addEventListener('click', (e) => {
     e.stopPropagation();
     winModal.style.display = "none";
+    document.querySelector('#timer p').textContent = '00:45';
     start.addEventListener('mouseleave', runGame);
 });
 
 loseModal.addEventListener('click', (e) => {
     e.stopPropagation();
     loseModal.style.display = "none";
+    document.querySelector('#timer p').textContent = '00:45';
     start.addEventListener('mouseleave', runGame);
 });
 
-//event listener on the start button to make splashscreen disappear
+// event listener on the start button to make splashscreen disappear
 document.querySelector('.startButton').addEventListener('click', () => {
     document.querySelector('#splashMain').style.display = 'none';
     document.querySelector('#gameScreen').style.display = 'block';
     start.addEventListener('mouseleave', runGame);
 })
 
-//listen for collisions with obstacles
+// listen for collisions with obstacles
 function listenForCollisions() {
     collision.forEach(item => {
         item.addEventListener('mouseenter', lose);
     });
 }
 
-//win if u get to Exit door (no timers yet)
+// win if you get to Exit door
```

Comment:
The names of these functions are pretty clear so comments not really needed. The name `collision` could be a bit more semantic - I wouldn't immediately know what a "collision" is from reading the code. I don't think you should change that at this point but I'd say remove the comments